### PR TITLE
Adding the Meshery project

### PR DESCRIPTION
**Pull request by leecalcote**

### CODE REVIEW

**Comment by leecalcote** [comment]
Meshery is comprised of a number of repositories under the layer5io org, which is why the URL for the "help wanted" label is complex and not passing. If it's not possible to use this type of URL, this can be updated to https://github.com/layer5io/meshery/labels/help%20wanted

**Comment by shiftkey** [comment]
@leecalcote thanks for the heads up - I'm gonna see if I can fix that hint to support org-wide cheks as part of reviewing this PR

There's also a change in here:

```
 1 files found in projects directory which are not YAML files:
 - _data/projects/meshery.yaml
```

I believe this needs to be `.yml` to work with Jekyll.

File: _data/projects/meshery.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- kubernetes
+- docker
+- prometheus
+- grafana
+- consul
+- performance
+- networking
+upforgrabs:
+  name: help wanted
+  link: https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+
```
Comment (shiftkey) [inline _data/projects/meshery.yaml]: I believe this is the root cause - the stray unicode character is throwing our JSONSchema validation.

```suggestion
  link: https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+
```

Comment (leecalcote) [reply _data/projects/meshery.yaml]: > @leecalcote thanks for the heads up - I'm gonna see if I can fix that hint to support org-wide cheks as part of reviewing this PR
> 
> There's also a change in here:
> 
> ```
>  1 files found in projects directory which are not YAML files:
>  - _data/projects/meshery.yaml
> ```
> 
> I believe this needs to be `.yml` to work with Jekyll.

Much appreciated, @shiftkey. Do let me know if there's any further information that I can provide or updates that I should make here. 👍

**Comment by leecalcote** [comment]
Regarding the [site](https://github.com/up-for-grabs/up-for-grabs.net/pull/2006/checks?check_run_id=622712134#step:4:264) build failure, this is a bit of a head-scratcher in that the `Meshery.yaml` is valid yaml.

**Comment by shiftkey** [comment]
> Regarding the [site](https://github.com/up-for-grabs/up-for-grabs.net/pull/2006/checks?check_run_id=622712134#step:4:264) build failure, this is a bit of a head-scratcher in that the `Meshery.yaml` is valid yaml.

@leecalcote let me confirm why we flag `.yaml` here as it gets a mention in [the Jekyll docs](https://jekyllrb.com/docs/datafiles/#the-data-folder) as being valid.

**Comment by ritwik12** [comment]
@leecalcote Ping!!

**Comment by shiftkey** [comment]
@leecalcote apologies for the delay - I'd fixed this in our tooling gem and not checked back in when it was bumped back in https://github.com/up-for-grabs/up-for-grabs.net/pull/2087

I merged the latest changes into this PR and we're good to land this. Thanks for your patience!

**Comment by leecalcote** [comment]
Oh, very good! Thanks much, @shiftkey. 👍

**Pull request merged**